kubernetes/CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+## Contributing
```

Comment:
Rather than contributing, could we please rename this to developer documentation?

Reply:
Sounds good. I will rename the file